package.json

```diff
@@ -7,7 +7,6 @@
     ],
     "packageManager": "yarn@3.6.0",
     "scripts": {
-        "postinstall": "husky install",
```

Comment:
lets add a ticket to add the yarn v3 plugin to replace this, this way other contributors other then us can have an easier time getting the repo setup on their machine.

Reply:
added a ticket to notion